utils/oscap.8

```diff
@@ -75,6 +75,11 @@ You may specify OVAL Definition files as the last parameter, XCCDF evaluation wi
 Select a particular profile from XCCDF document.
 .RE
 .TP
+\fB\-\-rule RULE\fR
+.RS
+Select a particular rule from XCCDF document. Only this rule will be evaluated.
```

Comment:
@mpreisler wasn't aware {{--profile}} is no longer necessary - still by specifying profile, we get values as specified in that particular profile, right? I am missing this (in my opinion quite useful) behaviour being pointed out.